doc/reference/tooling/tt_cli/install.rst

```diff
@@ -66,16 +71,42 @@ dependencies, such as a C compiler. Make sure they are available in the system.
 
 Tarantool Enterprise Edition is installed from prebuilt packages.
 
+Development versions
+~~~~~~~~~~~~~~~~~~~~
+
+``tt install`` can be used to build custom Tarantool and ``tt`` versions for
+development purposes. The sources for such version can be either stored locally
+or exist on GitHub (specific commits or pull requests).
+
+To build Tarantool from a local source, use the ``tarantool-dev`` program name and
```

Comment:
`tarantool-dev` does not build tarantool from local source, it just adds add a link to already built tarantool to the current tt environment:
./bin/tarantool -> tnt_source_dir/src/tarantool
The same is done for include dirs.

Reply:
Thanks for clarification. Rewrote the part about `tarantool-dev`.